lib/resources/okl_intrinsic_cuda.h

```diff
@@ -0,0 +1,30 @@
+namespace {
```

Comment:
Question. Why does every other include use `#pragma once`, But this one uses an anonymous namespace without `#pragma once` ?

Reply:
it's ebmedings that's why it's without `#pragram once`